packages/knex/src/query/QueryBuilderHelper.ts

```diff
@@ -459,6 +459,16 @@ export class QueryBuilderHelper {
         continue;
       }
 
+      if (cond[key][op] === null) {
+        if (replacement === QueryOperator.$ne) {
+          clause[m](this.knex.raw(`${this.knex.ref(this.mapper(key))} is not ?`, cond[key][op]));
+          continue;
+        } else if (replacement === QueryOperator.$eq) {
```

Comment:
there is no need for else branch
```suggestion
        }

        if (replacement === QueryOperator.$eq) {
```

Reply:
changed it to similar code from `where` conditions